hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/ITestApacheClientConnectionPool.java

```diff
@@ -18,6 +18,9 @@
 
 package org.apache.hadoop.fs.azurebfs.services;
 
+import java.io.IOException;
+
+import com.sun.tools.javac.util.Pair;
```

Comment:
use `org.apache.hadoop.util.functional.Tuples` to create  which will create an implementation of `Map.Entry<K, V>`; lets us move to the java 9 equivalent transparently

Reply:
Thanks! Taken.